src/sonic-yang-models/yang-models/sonic-head.yang

```diff
@@ -0,0 +1,101 @@
+module sonic-head {
```

Comment:
Curious, why is it called "sonic-head" instead of something like 'sonic-types' ?

Reply:
I am good with sonic-types, makes more sense. Note this module will not have top level container as defined in sonic guidelines, so better to have a separate file\module. Sonic-type is good.